commons-library/src/main/java/com/afh/skytouch/commons/dto/GenericProduct.java

```diff
@@ -0,0 +1,29 @@
+package com.afh.skytouch.commons.dto;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import java.time.Instant;
+import java.util.UUID;
+
+@Entity
+@Data
+public class GenericProduct
+{
+    @Setter(AccessLevel.NONE)
+    private UUID id;
+
+    private String Name;
+
+    @Setter(AccessLevel.NONE)
+    private Instant creationDate;
+
+    private String description;
+
+    {
```

Comment:
Any particular reason to have this initialisation as a static block of code and not directly in the declaration of the variable itself? Just curious...